src/feed_the_troll/servers.py

```diff
@@ -167,7 +167,9 @@ def _start_server(self, server_name, server_configuration):
         reconfigure_module = importlib.import_module(server_configuration['module'])
         reconfigure_server_namespace = namespace_from_configuration(server_name, server_configuration)
         if 'overrides' in server_configuration:
-            rospy.set_param(reconfigure_server_namespace, server_configuration['overrides'])
+            parameters = rospy.get_param(reconfigure_server_namespace)
```

Comment:
Nevermind, as long as the params get loaded in the same launch file as the feeder we should be save:
```
While roslaunch guarantees that all parameters are set prior to executing any nodes, 
there is no ordering when it comes to executing the nodes.
```
[source: Robots with ROS](https://books.google.co.kr/books?id=G3v5CgAAQBAJ&pg=PT505&lpg=PT505&dq=ros+launch+order+of+execution+parameters&source=bl&ots=P0Lj43gaIE&sig=5Flh6BmhQPDs3eNAGoYNpnAx5IY&hl=en&sa=X&ved=0ahUKEwikh5qMwdfRAhUFn5QKHeIhCxQQ6AEIODAF#v=onepage&q=ros%20launch%20order%20of%20execution%20parameters&f=false)